schema/show.js

```diff
@@ -34,10 +34,14 @@ import {
   GraphQLUnionType,
 } from 'graphql';
 
-const kind = ({ artists, fair }) => {
+const kind = ({ artists, fair, artists_without_artworks }) => {
   if (isExisty(fair)) return 'fair';
-  if (artists.length > 1) return 'group';
-  if (artists.length === 1) return 'solo';
+  if (artists.length > 1 || (artists_without_artworks && artists_without_artworks.length > 1)) {
+    return 'group';
+  }
+  if (artists.length === 1 || (artists_without_artworks && artists_without_artworks.length === 1)) {
+    return 'solo';
+  }
```

Comment:
I guess actually we could have an empty show, without any artists at all. Such a record _would_ be pathological but it'd be better if that maybe did return `undefined`? Rather than forcing it to be one of the three kinds.